src/pages/guides/index.md

```diff
@@ -251,7 +251,8 @@ This feature is scheduled to be rolled out on 10th February, 2022
 For a more robust and reliable verification, Adobe I/O Events adds below security validations for events delivered to your webhook. 
 
 - sending additional field of recipient_client_id as part of your event payload
-- signing the event payloads digitally and sending those digital signatures as request headers to your webhook
+- signing all the event payloads digitally using a fixed public/private key pair generated by Adobe I/O Events and sending those digital signatures as request headers to your webhook
+- sending the relative paths of public keys, which are served from our fixed adobe domain adobeioevents.com so that no one else can use their arbitrary public/private key pairs, as webhook request headers.
 
 I/O Events generates two public-private key pairs and sign your event payload using digital signatures following the steps below
```

Comment:
sign -> signs
I/O Events generates two public-private key pairs and **sign** your event payload using digital signatures following the steps below